compensation.adoc

```diff
@@ -40,7 +40,9 @@ This is what the GitHub issue is for. It should go in the https://github.com/bis
 [NOTE]
 .How do I know how much to request?
 ====
-Bisq contributions are valued _on completion_. The metric for that is _merged to master_ or otherwise _already delivered_ if the contribution is not code. How you determine the value you request is up to you, but it should be based on the value of the contribution you made, not on the raw time you spent.
+Bisq contributions are valued _on completion_. The metric for that is _merged to master_, or _already delivered_ if the contribution is not code. How you determine the value you request is up to you, but it should be based on the value of the contribution you made, not on the raw time you spent.
+
+A good rule of thumb, if you are unsure about the the value of your contribution: consider what you would charge for the work if you did it as a freelancer.
```

Comment:
"the the value" -> "the value"

Reply:
Yikes, thanks. Fixed with https://github.com/bisq-network/bisq-docs/pull/110.